src/systems/ec/implementations/vanilla/packages/ec-utility-colorize/ec-utility-colorize.scss

```diff
@@ -1,5 +1,7 @@
 /**
  * Colorize utilities
+ *
+ * Deprecated
```

Comment:
I think it will also be worth adding `[DEPRECATED] title...` to the README files